package_build_list_host_linux.json

```diff
@@ -6,7 +6,8 @@
     "build_from_source": {
         "assimp-5.1.6-rev1-linux":  "Scripts/extras/pull_and_build_from_git.py ../../package-system/assimp --platform-name Linux --package-root ../../package-system --clean",
         "AWSGameLiftServerSDK-3.4.2-rev1-linux": "package-system/AWSGameLiftServerSDK/build_package_image.py --platform-name linux",
-        "AWSNativeSDK-1.9.50-rev1-linux": "Scripts/extras/pull_and_build_from_git.py ../../package-system/AWSNativeSDK --platform-name Linux --package-root ../../package-system --clean",
+        "AWSNativeSDK-1.9.50-rev2-linux-openssl1": "Scripts/extras/pull_and_build_from_git.py ../../package-system/AWSNativeSDK --platform-name Linux-OpenSSL1 --package-root ../../package-system/AWSNativeSDK/temp --clean",
+        "AWSNativeSDK-1.9.50-rev2-linux-openssl3": "Scripts/extras/pull_and_build_from_git.py ../../package-system/AWSNativeSDK --platform-name Linux-OpenSSL3 --package-root ../../package-system/AWSNativeSDK/temp --clean",
```

Comment:
The 'L' being next to '1' makes it hard to read.
Could you put a a dash after openssl and the version number?